Makefile

```
@@ -36,13 +55,20 @@ container-image: ## Build container image
 binary-snapshot: ## Build binary snapshot
 	goreleaser release --rm-dist --skip-publish --snapshot
```

change into to use with the latest version (new version works smoother than the previous version)
```suggestion
	VERSION=v${GORELEASER_VERSION} goreleaser release --clean --skip=publish --snapshot
```